esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -1843,45 +1843,60 @@ static StringBuffer &getDropZoneHost(const char *planeName, IPropertyTree *plane
     return host;
 }
 
+void CFileSprayEx::throwOrLogDropZonePlaneNotFound(const char* host, const char* path)
+{
+    if (isContainerized())
```

Comment:
I almost put a comment in last review to say, there are times when conditional compile on macro (_CONTAINERIZED) makes sense, The code below being one of them, which will not compile in containerized...